.github/workflows/bump-app-version.yml

```diff
@@ -37,5 +38,5 @@ jobs:
       - uses: actions/github-script@v5
         with:
           script: |
-            const script = require('.github/scripts/automergeVersionBump.js')
+            const script = require('.github/scripts/automergePR.js')
```

Comment:
nit: what do you think of explicitly passing the expected files as argument instead of an env variable?

Reply:
yes this is a much better idea...i have updated to pass an argument through for this script